instruments/toptica/topmode.py

```diff
@@ -224,7 +239,14 @@ def lock_start(self):
             :return: The datetime of start of mode-locking for specified laser
             :type: `datetime`
             """
-            return ctdate(self.parent.reference(self.name + ":charm:reg:started"))
+            if not self.is_connected:
+                return
```

Comment:
If the laser is not connected, I don't think that we should be silently exiting this call. Someone's code could depend on this returned datetime value, but if we're just returning `None` then they could have problems down the line when debugging.